PatientRegistrationApp/src/App.vue

```diff
@@ -1,40 +1,40 @@
 <template>
-  <v-app id="Patient-registration"> <!--style="background: #F5F5F5"--> 
+  <v-app id="Patient-registration"> 
     <v-main>
 		<v-container>
-			<v-card flat> 
-				<v-stepper v-model="page" class="elevation-0">	
+			<v-card class="elevation-12" min-width="400" max-width="1000"> 
+				<v-stepper v-model="page" class="elevation-0">
 					<v-toolbar flat color="primary" dark>
 						<!-- We could make the following toolbar dynamic, but for now I just have one title (defined at the end of this file)-->
-						<v-toolbar-title>{{title}}</v-toolbar-title>
+						<v-spacer /><v-toolbar-title style="font-size:2em" class="font-weight-bold">{{title}}</v-toolbar-title> <v-spacer />
```

Comment:
Were you adjusting font sizes in this card as well? It looks like so in multiple places....